VSEmbed.DemoApp/App.config

```diff
@@ -181,6 +181,14 @@
         <assemblyIdentity name="System.Threading.Thread" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
         <bindingRedirect oldVersion="0.0.0.0-4.0.1.0" newVersion="4.0.1.0" />
       </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.VisualStudio.Shell.10.0" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
+        <bindingRedirect oldVersion="2.0.0.0-14.0.0.0" newVersion="15.0.0.0" />
+      </dependentAssembly>
+      <dependentAssembly>
+        <assemblyIdentity name="Microsoft.VisualStudio.Shell.Immutable.10.0" publicKeyToken="b03f5f7f11d50a3a" culture="neutral" />
```

Comment:
This forces Roslyn to load v15.0 of `Shell.Immutable.10` which type forwards `SVsServiceProvider` to Shell.Framework.